news/2024/2024-11-17-kel-lan-tournament-3.md

```diff
@@ -0,0 +1,81 @@
+
+Qualifiers will take place online between **4th–18th December**.
+
+**If you signed up for the tournament, you need to join the [Discord](https://discord.com/invite/TSgMXsaZRR)!**
+
+This 2-day event during **February 8th-9th** features a major tournament alongside bundle of side events, all streamed from the [KEL3](https://www.twitch.tv/kinguin_lounge) Twitch channel!
+
+## Schedule
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| **Registrations** | October 8th - December 2nd 2024 |
+| **Qualifiers** | December 4th - December 18th 2024 |
+| **Day 1** | February 8th 2025 (10:00 - 18:00) |
+| **Day 2** | February 9th 2025 (10:00 - 18:00) |
+
+## Prizes
+
+The total prize pool of **610 PLN** will be awarded to the **top 3 players**, along with exciting physical prizes, including the **Wooting UwU Keypads**!
+
+| Placement | Prizes |
+| :-: | :-- |
+| 1st | **Wooting UwU RGB Stealth Keypad** (courtesy of Wooting), 48% of the prizepool,  profile badge (pending) |
+| 2nd | **Wooting UwU Keypad** (courtesy of Wooting), 32% of the prizepool |
+| 3rd | 16% of the prizepool |
+
+## 1v1 double-elimination tournament
+
+The stage is set for an exciting showdown where every match counts. In this fun and fierce 1v1 double-elimination tournament, players will face off in a battle of skill, with every click adding to the excitement and bringing them one step closer to victory!
+
+Below is the bracket schedule:
+
+| Day | Brackets |
+| --: | :-- |
+| **Day 1** | Round of 32, Round of 16, Quarterfinals |
+| **Day 2** | Semifinals, Finals, Grand Finals |
+
+## Side events
+
+During the breaks between or after tournament matches, we'll be hosting a variety of fun side events. Stay tuned for more details coming your way soon!
+
+## Special guests
+
+Wondering who'll be joining the action? Take a look at our first guest reveal:
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/videonotdoneyet" frameborder="0" allowfullscreen></iframe>
+</div>
+
+And that's just the beginning—more special guests will be announced soon! Stay tuned on our [**Discord**](https://discord.com/invite/TSgMXsaZRR) for all the latest updates!
+
+## Participants
+
+This year's tournament brings an exciting twist—special guests will be joining the competition! Qualify, and you'll have the chance to go head-to-head with top players and special guests alike.
+
+It's more than just a tournament; it's an opportunity to prove yourself alongside some of the most respected names in the community.
+Ready to rise to the challenge?
+
+---
+
+For all the finer details, check out the [**forum post**](https://osu.ppy.sh/community/forums/topics/1987674?n=1) and be sure to join our [**Discord Server**](https://discord.com/invite/TSgMXsaZRR) to stay up-to-date with the latest announcements, guest reveals, and event surprises.
+
+See you at the event!
+
+—myksmyks & flapczek
```

Comment:
i usually like to sort them A-Z
```suggestion
—flapczek & myksmyks
```

Reply:
Placement like that was intentional as i was the one that started the whole thing and flapczek joined me for the 3rd iteration, so if possible would rather to make it stay with me first (small things count ;p)

Reply:
@Walavouchey can an exception be made here?

Reply:
it's not like there was a rule. whatever makes sense or sounds best is how i've thought of it

Reply:
feel free to dismiss this then